doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -958,6 +961,19 @@ If enabled, nodes will be *stopped* when the cluster scales down. If disabled, n
 * **Type:** Boolean
 * **Default:** ``True``
 
+.. _cluster-configuration-use-internal-ips:
+
+``provider.use_internal_ips``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If enabled, Ray will use private IP addresses for communication between nodes.
+This should be omitted if your network interfaces use public IP addresses.
+
+* **Required:** No
+* **Importance:** Low
```

Comment:
Not sure about importance.

Reply:
Hmmm can we be a bit more explicit about which features will use private instead of public ip? 

For example, does this mean that calling `ray up` would have to be done from a machine which can access the private ip's of the cluster?

Will the nodes in the cluster have public ip addresses still that are just unused (particularly relevant for accessing the dashboard/other head node services).

Reply:
Added more, @wuisawesome PTAL